notebooks/algorithms/link_prediction/Jaccard-Similarity.ipynb

```diff
@@ -8,47 +8,16 @@
     "# Jaccard Similarity\n",
     "----\n",
     "\n",
-    "In this notebook we will explore the Jaccard vertex similarity metrics available in cuGraph. cuGraph supports:\n",
```

Comment:
The "cugraph supports:" section in this notebook was referring to weighted Jaccard, which isn't being shown in the notebook anymore, so I just took it out to keep it simple.  I could mention that weighted Jaccard is supported though and that it's currently not being demonstrated here.

FYI: weighted Jaccard was removed because we no longer support the old vertex weight approach, which the notebook was still trying to show (i.e. the notebook was broken).  We instead support edge weights now, but I don't have an example of a graph with unique edge weights to properly demonstrate this.

Re: history - I was wondering if we could discuss this.  Are these tables needed?  They seem like a maintenance burden that doesn't add value (git/github tracks revision history much better), and I don't see any other notebooks in the community or even RAPIDS doing this.  For example, [here's a cudf notebook](https://github.com/rapidsai/cudf/blob/ac60656bc929c08f522d23328a515caa52cee989/docs/cudf/source/user_guide/10min.ipynb) that just looks so much easier to read.  I do think if we document timing numbers we should document the hardware used, but this notebook isn't doing that either.